services/net/notification/Config/NotificationOptions.cs

```diff
@@ -19,24 +19,5 @@ public class NotificationOptions : ServiceOptions
     /// </summary>
     public int AlertId { get; set; }
 
```

Comment:
You could add a Reporting section in this config if you didn't want to change the dependency injection configuration.

Reply:
I did think of that, but my thought was that then the ENV style path to the variables would be different to the way it's set up in the Reporting Service. 

For example in the Reporting Service the path would be `Reporting__AddToReportUrl` but if we nested it in the `NotificationOptions` It would become `Service__ReportingOptions__AddToReportUrl`. If we can keep them the same, the possibly the DeploymentConfigs will be easier to keep in line.